readme.md

```diff
@@ -11,7 +11,7 @@ It's a fancy name for a simple concept: Extending a model (usually to add specif
 ## Installation
 
 ```bash
-composer require "calebporzio/parental=0.9"
+composer require "calebporzio/parental=0.10"
```

Comment:
Once 0.11 is released this becomes a problem again, I'd recommend changing this to:

```
composer require calebporzio/parental
```

No quotes needed and composer takes the latest version automatically. 🍰